__tests__/validations/component.test.tsx

```diff
@@ -941,12 +941,12 @@ test('Success when the reconfigure configuration optioms of the prefabs are vali
               type: 'PROPERTY',
               configuration: {
                 allowedKinds: ['TEXT', 'URL'],
-                allowManageValues: true,
                 createNewProperty: {
                   type: 'TEXT',
-                  dependsOn: 'model',
                   value: 'New property',
                 },
+                schemaModel: 'model',
```

Comment:
Will `schemaModel` replace `dependsOn` by default now?

Reply:
Yes, this is the schema model we need for creating/selecting the object property. And dependsOn is the dependent model option.

Reply:
I removed the schema model key, because we know the schema model from the selected property or the parent questionWidget